src/routers/pscTypeRouter.ts

```diff
@@ -20,9 +20,18 @@ router.get(Urls.PSC_TYPE, authenticate, handleExceptions(async (req: Request, re
 
 router.post(Urls.PSC_TYPE, authenticate, handleExceptions(async (req: Request, res: Response, _next: NextFunction) => {
     const lang = selectLang(req.body.lang);
+    const selectedType = req.body.pscType;
 
-    const nextPageUrl = getUrlWithTransactionIdAndSubmissionId(PrefixedUrls.INDIVIDUAL_PSC_LIST, req.params.transactionId, req.params.submissionId);
+    const nextPageUrl = getUrlWithTransactionIdAndSubmissionId(selectPscType(selectedType), req.params.transactionId, req.params.submissionId);
     res.redirect(addSearchParams(nextPageUrl, { lang }));
 }));
 
+const selectPscType = (pscType: any): string => {
+    switch (pscType) {
+    case "individual": return PrefixedUrls.INDIVIDUAL_PSC_LIST;
+    case "rle": return PrefixedUrls.RLE_LIST;
+    default: return PrefixedUrls.INDIVIDUAL_PSC_LIST;
```

Comment:
The default case (neither option selected) will get error handling later, so best add a TODO comment as reminder.